package.json

```diff
@@ -49,6 +49,7 @@
     "@fortawesome/react-fontawesome": "^0.1.11",
     "carbon-components-react": "^7.25.0",
     "d3": "^5.16.0",
+    "import-map-overrides": "^2.3.1",
```

Comment:
Why did you add this dependency?
Should be removed

Reply:
Sure. Will remove them.